workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/InviteMemberModal/InviteMemberModal.tsx

```diff
@@ -0,0 +1,309 @@
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import ClayModal, { useModal } from '@clayui/modal';
+import ClayForm, { ClayInput, ClayCheckbox } from '@clayui/form';
+import { useEffect, useState } from 'react';
+import { Liferay } from '../../liferay/liferay';
+import './inviteMemberModal.scss'
+
+interface InviteMemberModalProps {
+  handleClose: () => void;
+  selectedAccount: Account;
+}
+
+interface CheckboxRole {
+  roleName: string;
+  isChecked: boolean;
+}
+
+export function InviteMemberModal({
+  handleClose,
+  selectedAccount,
+}: InviteMemberModalProps) {
+  const { observer, onClose } = useModal({
+    onClose: () => handleClose(),
+  });
+
+  const [email, setEmail] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
```

Comment:
Please type it. You can also use something like:
```suggestion

 const userDataObj = {
   email: "",
   firstName: "",
   lastName: ""
  }

  const [userData, setUserData] = useState<UserDataType>(userDataObj);
```

Take a look [here](https://react.dev/learn/updating-objects-in-state)

Reply:
Thanks, i didn't know that i could use in that way